tslint.json

```diff
@@ -22,6 +21,11 @@
                 "allowSingleLine": true
             }
         ],
+        "interface-name": [
```

Comment:
Nu forceert hij dat alle interfaces zonder `I` prefix zijn ("`IUser`" geeft nu een error).
Wat mij betreft droppen we deze hele rule, het maakt mij niet zo uit wat de naamgeving van interfaces is.

Reply:
Doe maar I prefix forceren, dan vermijd je conflicten met component names en heb je meer consistentie en duidelijk herkenbare interface names.

Reply:
Is goed!